packages/features/portfolio/src/containers/portfolio-state-zero/hooks/use-card-data.tsx

```diff
@@ -95,16 +104,26 @@ export const useCardData = (
 
   const leveragedData = [
     {
-      accentTitle: <FormattedMessage defaultMessage={'Incentivized Yield'} />,
+      accentTitle: <FormattedMessage defaultMessage={'Incentivised Yield'} />,
```

Comment:
`Incentivised` with an "s" is the British spelling and we won that war

Reply:
vs code thinks the z version is a error